internal/controller/vmfilerestore_controller.go

```
@@ -0,0 +1,420 @@
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"strconv"
+
+	"github.com/go-logr/logr"
+	"github.com/sirupsen/logrus"
+	"golang.org/x/exp/maps"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
```

golangci-lint (2.5.0)

[error] File is not properly formatted

(gci)